src/main/java/app/controllers/UserController.java

```diff
@@ -0,0 +1,65 @@
+package app.controllers;
+
+import app.entities.User;
+import app.exceptions.DatabaseException;
+import app.persistence.ConnectionPool;
+import app.persistence.UserMapper;
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+
+public class UserController {
+    public static void addRoutes(Javalin app, ConnectionPool connectionPool) {
+        app.post("login", ctx -> login(ctx, connectionPool));
+        app.get("logout", ctx -> logout(ctx));
+        app.get("createuser", ctx -> ctx.render("createuser.html"));
+        app.post("createuser", ctx -> createUser(ctx, connectionPool));
+    }
+
+    private static void createUser(Context ctx, ConnectionPool connectionPool) {
+        String username = ctx.formParam("username");
+        String password1 = ctx.formParam("password1");
+        String password2 = ctx.formParam("password2");
+
+        if (password1.equals(password2)) {
+            try {
+                UserMapper.createuser(username, password1, connectionPool);
+                ctx.attribute("message", "Du er hermed oprettet med brugernavn: " + username +
```

Comment:
Change user created message to not mention a username as the user was created using an email